helm/grafana/templates/cnpg/scheduled-backup.yaml

```diff
@@ -0,0 +1,18 @@
+{{- if and (.Values.postgresqlCluster.enabled) (.Values.postgresqlCluster.backup.enabled) }}
+apiVersion: postgresql.cnpg.io/v1
+kind: ScheduledBackup
+metadata:
+  labels:
+    {{- include "grafana.labels" . | nindent 4 }}
+  name: {{ .Values.postgresqlCluster.name }}-hourly-backup
+  namespace:  {{ .Release.Namespace }}
+spec:
+# At 0 minute and 0 second of every hour
```

Comment:
I think this comment no longer make sense. as its already added where the value is defined.